### DISCUSSION TITLE
Timeseries panel: Option to use query id in fill below to

### DISCUSSION THREAD

**Discussion by schmitzi89** [grafana/grafana#37199] [Dashboards, Panels & Visualizations]
In my database I have multiple metrics from temperature sensors and a target temperature.
I display two trends in my dashboard: A Temperature metric and the target temperature.
I want to apply the override "fill below to" on the target temperature and select a query.

Currently I can only select the name of the temperature sensor (lets say Temperature 01). 
But if I switch to temperature sensor 02 in the query the fill below doesnt work any more. It is lost.

It would be nice if the fill below could still work if I change the temperature signal in the query.

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). Due to the age and number of responses to this discussion, we are deciding to close it. If this is something you would like to see in Grafana, feel free to open an [issue](https://github.com/grafana/grafana/issues) so the discussion can continue. Thank you!
